specification/components/stu3/schemas/endpoints/a024-retrieve-advice-and-guidance.yaml

```diff
@@ -36,6 +43,16 @@ description: |
   convert the advice and guidance request into an appointment request. 
   If authorised, [[HYPERLINK_A027]] can be used to convert the advice and guidance request into an appointment request.
 
+  An advice and guidance request may have been converted to a referral (with the same UBRN), 
+  and in addition a referral pathway in e-RS can be made up of more than one UBRN. As such, 
+  it is important that all clinical information is obtained across all the related UBRNs in the pathway. 
+  These may be identified using [[HYPERLINK_A005]] and will be returned if the user is authourised to view this.
+
+  If the converted advice and guidance request is referred to a service outside of your organisation, 
+  the subsequent referral information may not be accessible due to existing data-access rules. However, 
+  the advice and guidance summary ([[HYPERLINK_A024]]), attachments ([[HYPERLINK_A006]]), advice and guidance conversation ([[HYPERLINK_A025]]) and 
+  clinical information ([[HYPERLINK_A043]]) which took place before such a conversion still forms part of the clinical information and can still be retrieved.
+
   ## Related endpoints
 
   After using this endpoint, use [[HYPERLINK_A025]] to retrieve the Advice and Guidance conversation itself.
```

Comment:
I'm assuming the info about A043 in the Related endpoints is still required even though it's not mentioned on ERSSUP-63512?

Reply:
I wasn't sure it should be removed, I was more thinking it was missing from the story?